docs: add images for seq diagrams and secrets location in front end (#50)
CS3219-SE-Principles-and-Patterns/cs3219-project-ay2122-2122-s1-g9: frontend/README.md

1. Copy the `.env.example` file and rename it to `.env`

> 2. Update the variables in the `.env` file with the relevant Firebase keys. Set REACT_APP_USE_EMULATOR to true or false depending on which instance you wish to connect to.

Reviewer: ```suggestion
2. Update the variables in the `.env` file with the relevant Firebase keys. Set REACT_APP_USE_EMULATOR to any string if you need to connect to the emulators.
``` 
Not sure if this line should be changed 🤔 , even if you put `false`, the` REACT_APP_USE_EMULATOR` will be considered `true` as I did not put special logic to handle string conversion to boolean.
